### PR TITLE
feat: active mints carousel

### PR DESCRIPTION
**Pull request by lauchness**
## Description
This PR adds the active mints carousel to the landing page

Screen Shot 2023-08-17 at 2 18 34 PM
Screen Shot 2023-08-17 at 2 19 43 PM

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -133,7 +136,7 @@ const Home = async ({ searchParams }: Props) => {
             <div className="flex justify-between mb-4">
               <div className="flex flex-col [@media(min-width:724px)]:flex-row gap-4 items-start sm:items-center">
                 <div className="flex gap-4 items-center">
-                  <div className="flex justify-center items-center h-[64px] w-[64px] rounded-2xl bg-[#FF7DCB]">
+                  <div className="flex justify-center items-center h-[64px] w-[64px] rounded-2xl bg-ocs-pink">
```
Comment (gio-m-cb) [inline src/app/page.tsx:139]: If this is part of our palette, it should be defined in a theme config?`

**Review by gio-m-cb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for gio-m-cb @ 2023-08-17 17:22:47 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

File: src/app/page.tsx (hunk 1 above)
Comment (lauchness) [reply src/app/page.tsx:139]: It is - in tailwind.config.js

I'm changing this to use the already existing pallette

Comment (gio-m-cb) [reply src/app/page.tsx:139]: Sorry, I read this the same as here https://github.com/base-org/onchainsummer.xyz/pull/307/files#diff-73d7a23e5015801b9bcc9db601d6ec9594d3eb34e5bb23154e0ae4b0c30f1a3bR55. Should that be added as well?

Comment (lauchness) [reply src/app/page.tsx:139]: gotcha - yeah I'm reconciling those in this PR

https://github.com/base-org/onchainsummer.xyz/pull/308

And then I'm going to reconcile all the uses of `[#EFEFEF]` and `[#444]` in another PR upcoming

**Comment by wilsoncusack** [comment]
something up here, not sure if cause rebase needed 
image

**Comment by wilsoncusack** [comment]
@lauchness still seeing this 
image


also I think let's go most recent left to right, rather than oldest first

**Comment by lauchness** [comment]
@wilsoncusack rebased and sorted appropriately.

The issue with the address pill is the one I addresses yesterday, where if the description isn't present, the address pill is not vertically aligned.

I think I can fix this, but it's going to take some refactoring of the DropCard and Carousel.

**Review by AshishYelekar — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for AshishYelekar @ 2023-08-18 15:49:32 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**